roles/auth_prepare/tasks/auth_prepare.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Task to check if Auth is already setup
+
+- name: Check service is already exists
+  shell: "/usr/lpp/mmfs/bin/mmuserauth service check | grep -q 'File configuration is *'"
+  register: service_result
+  ignore_errors: yes
+  failed_when: service_result.rc == 2
+  when: is_admin_node | default(false) == true
```

Comment:
same is_admin_node not required , we should run these task from one of the CES node

Reply:
Resolved